index.html

```diff
@@ -1213,6 +1213,16 @@ <h4>Internal slots for ConsumedThing</h4>
           <td>`null`</td>
           <td>The <a>Thing Description</a> of the {{ConsumedThing}}.</td>
          </tr>
+        <tr>
+          <td><dfn>[[\activeSubscriptions]]</dfn></td>
+          <td>`{}`</td>
```

Comment:
It was a good idea to have an internal slot as a set, as it is usable with all policies (replace existing, reject if existing, add, ...).